README.md

```diff
@@ -1 +1,49 @@
-# tdd
\ No newline at end of file
+# tdd
+
+## **Project Description**
+
+> In this project we implemented a class with some methods, but we did it by doing test-driven development (TDD). Remember that the idea is to write tests first and then the code. This will be verified in your commits history, so it is important to use the proper sequence.
+
+## Built With
+
+`Ruby`
+
+`TDD`
+
+## Getting Started
+
+- Make sure you have Ruby installed in your system. You can install it here.
+- Clone this repository by running `https://github.com/Ugboaja-Uchechi/tdd.git` in your command line.
+- Navigate to the repository by running cd `tdd`.
+- Run the project by running ruby person.rb on the command line.
+- To run the tests make sure you install rspec gem by running gem install rspec on the command line then run `rspec spec` on the command line to see the test passing
+
+## Authors
+
+### Author 1
+
+👤 **Ugboaja Uchechi**
+
+- [Github](https://github.com/Ugboaja-Uchechi)
+
+- [LinkedIn](https://www.linkedin.com/in/stephanie-ugboaja-930a2a216/)
+
+### Author 2
+
+- GitHub: [@yaseerokino](https://github.com/yaseerokino)
+
+- LinkedIn: [Yaseer Okino](https://linkedin.com/in/yaseerokino)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] the link to the License is not working, please make sure the link is working as shown in [the Microverse template](https://github.com/microverseinc/readme-template).